h2o-py/tests/pyunit_utils/utils_for_glm_tests.py

```diff
@@ -0,0 +1,150 @@
+import h2o
+from h2o.estimators import H2OGeneralizedLinearEstimator as glm
+from h2o.exceptions import H2OValueError
+from h2o.grid.grid_search import H2OGridSearch
+
+def gen_constraint_glm_model(training_dataset, x, y, solver="AUTO", family="gaussian", linear_constraints=None, 
+                             beta_constraints=None, separate_linear_beta=False, init_optimal_glm=False, startval=None,
+                             constraint_eta0=0.1258925, constraint_tau=10, constraint_alpha=0.1, 
+                             constraint_beta=0.9, constraint_c0=10):
+    """
+    This function given the parameters will return a constraint GLM model.
+    """
+    if linear_constraints is None:
+        raise H2OValueError("linear_constraints cannot be None")
+        
+    params = {"family":family, "lambda_":0.0, "seed":12345, "remove_collinear_columns":True, "solver":solver, 
+              "linear_constraints":linear_constraints, "init_optimal_glm":init_optimal_glm, 
+              "constraint_eta0":constraint_eta0, "constraint_tau":constraint_tau, "constraint_alpha":constraint_alpha,
+              "constraint_beta":constraint_beta, "constraint_c0":constraint_c0}
+    if beta_constraints is not None:
+        params['beta_constraints']=beta_constraints
+        params["separate_linear_beta"]=separate_linear_beta
+    if startval is not None:
+        params["startval"]=startval
+        
+    constraint_glm = glm(**params)
+    constraint_glm.train(x=x, y=y, training_frame=training_dataset)
+    return constraint_glm
+
```

Comment:
```suggestion


```
Ideally, I would try to stick to PEP-8 and [keep 2 blank lines between top-level functions](https://peps.python.org/pep-0008/#blank-lines).

Reply:
GAM, model selection, anovaGLM all have coefficients.  I just have not come around and add them yet.